com.unity.netcode.gameobjects/Documentation~/encryption.md

```diff
@@ -0,0 +1,47 @@
+
+## Using DTLS encryption
```

Comment:
maybe a stupid question but I was curious whether or not we could use browser's certificates somehow instead of baking our very own certificates into the build? perhaps with a .jslib? @simon-lemay-unity

Reply:
We are definitely **not** baking our very own certificates into the build.

On WebGL, the server cert will be signed by an authority that the browser accepts. I do not know if it is setup so that the server provides the public part and the client accepts it as long as it is properly signed authoritatively, or it the client certificate part is actually needed. but this is a moot question anyway, because...

On regular platforms, the client certificate will be bundled in the game by the game team. They can use whatever they want for the server certificate (private key) as long as the client part (public key) matches. So, looking at the code you can see the client part provides the same hostname and client certificate pair. That makes the code uniform and simpler.